RASP_support/REPL.py

```diff
@@ -496,11 +461,11 @@ def nice_matrix_line(m):
 if __name__ == "__main__":
 	REPL().run()
 
-def runner():
-	a = REPL()
-	try:
-		a.run()
-	except Exception as e:
-		print(e)
-		return a,e
-	return a,None
\ No newline at end of file
+#def runner():
+#	a = REPL()
+#	try:
+#		a.run()
+#	except Exception as e:
+#		print(e)
+#		return a,e
+#	return a,None
```

Comment:
I'd rather leave this uncommented (this is how I run the REPL in python, so I can poke around the environment and errors if something is being weird), can rename it to debug_runner if want to be explicit though

Reply:
makes sense